cypress/plugins/addAuth.js

```diff
@@ -0,0 +1,21 @@
+const fs = require("fs");
+const os = require("os");
+const path = require("path");
+const axios = require("axios");
+
+axios
+  .post("https://wroom.io/authentication/signin", {
+    email: "cypress@prismic.io",
+    password: "AhK9yohhie9ahyohn3w",
```

Comment:
These should be an env var, we can ask mehdi and michel to and them to the repository settings, and a check to let the user know to set them when running locally.

Reply:
yes true, will ask them